modules/google-analytics/wp-google-analytics.php

```diff
@@ -0,0 +1,615 @@
+<?php
+/**
+ * Plugin Name: WP Google Analytics
+ * Plugin URI: http://bluedogwebservices.com/wordpress-plugin/wp-google-analytics/
+ * Description: Lets you use <a href="http://analytics.google.com">Google Analytics</a> to track your WordPress site statistics
+ * Version: 1.4.0
+ * Author: Aaron D. Campbell
+ * Author URI: http://ran.ge/
+ * License: GPLv2 or later
+ * Text Domain: wp-google-analytics
+ */
+
+define('WGA_VERSION', '1.4.0');
```

Comment:
This should probably be made unique to avoid conflicts with the existing plugin.